isoimage.py

```diff
@@ -0,0 +1,151 @@
+import yaml
```

Comment:
The script itself appears to be functioning as designed. I did some rudimentary smoke testing. Good job!
```
codz@dt:~/projects/piedpiper-pakman-faas$ \
comm -3 \
  <(cd ~/projects/piedpiper-pakman-faas/target/; find ./ | sort) \
  <(cd /mnt/iso; find ./ | sort)
```
Looks like the output in `target/` properly matches the contents of the ISO. Also looks like the mock data combined with the instructions in your `sample.yaml` yield the expected results:

```
codz@dt:~/projects/piedpiper-pakman-faas$ tree /mnt/iso
/mnt/iso
├── bin
│   ├── binary1.zip
│   ├── binary2.tar
│   └── deps
├── docs
│   ├── Itinerary_ Flight to New Orleans.pdf
│   ├── Itinerary_ Raleigh.pdf
│   ├── logs
│   ├── __MACOSX
│   ├── Notification of Foreign Travel.doc
│   ├── Receipt for Flight to New Orleans.pdf
│   └── Receipt for Raleigh.pdf
├── README.MD
└── src
    └── test_tar
        ├── Certificate (1).pdf
        ├── Certificate (2).pdf
        ├── Certificate (3).pdf
        └── Certificate.pdf
```
Looks like it is still lacking some error handling in places, for instance, I didn't create the file called `ci-cd_pipeline_output.log` in my `logs/` directory initially, and it quietly skipped it. When the user explicitly requests a file by name, it might be best to throw an exception when nothing is present in those cases. Or, it might even be nice to let the user specify whether or not is allowed to match nothing, even for regex. Thoughts @ragingpastry ?

Reply:
added new function validate_yaml_file(data_loaded) that verifies all the files in the yaml status and exits if it does not find one in this push 97c1590